codelab-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+FROM gcr.io/kubeflow-images-public/tensorflow-1.8.0-notebook-cpu
```

Comment:
Can you add a Makefile like:
https://gist.github.com/jlewi/5d323e483bf386b774130f063e683b33

Importantly we'd like to include the git hash of the code in the image metadata.

Reply:
Added.